src/middlewared/middlewared/plugins/alert.py

```diff
@@ -0,0 +1,444 @@
+from collections import defaultdict
+import copy
+from datetime import datetime
+import json
+import os
+import traceback
+
+from freenasUI.support.utils import get_license
+from licenselib.license import ContractType
+
+from middlewared.alert.base import *
+from middlewared.alert.base import AlertService as _AlertService
+from middlewared.schema import Dict, Str, Bool, Int, accepts, Patch
+from middlewared.service import (
+    ConfigService, CRUDService, Service, ValidationErrors,
+    periodic, private,
+)
+from middlewared.utils import load_modules, load_classes
+
+POLICIES = ["IMMEDIATELY", "HOURLY", "DAILY", "NEVER"]
+DEFAULT_POLICY = "IMMEDIATELY"
+
+alert_sources = []
+alert_services_factories = {}
```

Comment:
I have a hard time seeing these vars within the methods and realizing they are global, what do you think about making it uppercase?